src/libs/TransactionUtils.ts

```diff
@@ -126,6 +128,8 @@ function buildOptimisticTransaction(
         filename,
         category,
         tag,
+        taxCode,
+        taxAmount,
```

Comment:
This is causing the deploy blocker: https://github.com/Expensify/App/issues/42114#issuecomment-2109081369